webserver/server/app/controllers/ProjectedNftRangeController.ts

```diff
@@ -38,11 +38,11 @@ export class ProjectedNftRangeController extends Controller {
                 actionTxId: data.action_tx_id,
                 actionOutputIndex: data.action_output_index,
                 asset: data.asset,
-                amount: parseInt(data.amount),
```

Comment:
does this work? BigInt is not JSON serializable in Javascript

Reply:
let's just use string for large ints